partner_credit_limit/models/account_invoice.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import models, api, _
+from odoo import exceptions
+
+
+class AccontInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    @api.multi
+    def check_payment_type(self):
+        if self.filtered(lambda invoice:
```

Comment:
What about if there is an invoice with payment_type != credit and another one with payment_type = credit?